gensim/models/word2vec.py

```diff
@@ -1573,18 +1573,19 @@ def accuracy(self, questions, restrict_vocab=30000, most_similar=most_similar, u
         The accuracy is reported (=printed to log and returned as a list) for each
         section separately, plus there's one aggregate summary at the end.
 
-        Use `restrict_vocab` to ignore all questions containing a word whose frequency
-        is not in the top-N most frequent words (default top 30,000).
+        `restrict_vocab` is an optional integer which limits the vocab to be used
+        for answering questions. For example, restrict_vocab=10000 would only check
+        the first 10000 word vectors in the vocabulary order. (This may be meaningful
+        if you've sorted the vocabulary by descending frequency.)
```

Comment:
Please add a comment to what happens for `case_insensitive` and `restrict_vocab` together.